contracts/DepositContract.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity ^0.4.22;
+
+contract DepositContract {
+
+  mapping(address => uint) private Stakes;
+  mapping(address => uint) private Indices;
+  address[] private nodeAddressList;
+
+  event LogNewNode(address indexed nodeAddress, uint index, uint stake);
+  event LogUpdateStake(address indexed nodeAddress, uint index,  uint stake);
+  event LogDeleteNode(address indexed nodeAddress, uint index);
+  
+  function isNode(address nodeAddress) public constant returns(bool isIndeed) 
```

Comment:
Why do we need to track node is deposit contract? I think it's better to be simple, just tracking deposits.

Reply:
The node here is just the node address.

Reply:
Changed the name.